docs/dev/setup/jenkins-gitlab.rst

```diff
@@ -770,16 +775,16 @@ JUnit formatted results to any URL.
 You can download the current release of the plugin
 `here <https://github.com/ls1intum/jenkins-server-notification-plugin/releases>`__
 (Download the **.hpi** file). Go to the Jenkins plugin page (*Manage
-Jenkins → Manage Plugins*) and install the downloaded file under the
-*Advanced* tab under *Upload Plugin*
+Jenkins → Plugins*) and install the downloaded file under the
+*Advanced* tab under *Deploy Plugin*
 
 .. figure:: setup/jenkins-gitlab/jenkins_custom_plugin.png
    :align: center
 
 Jenkins Credentials
 """""""""""""""""""
 
-Go to *Manage Jenkins -> Security -> Manage Credentials → Jenkins → Global credentials* and create the
+Go to *Manage Jenkins → Security → Manage Credentials → Jenkins → Global credentials* and create the
```

Comment:
I only skimmed the parts of the setup documentation that were not modified in the PR and may have overlooked something, which may be why there is no _Manage Credentials_ in my local setup under  _Manage Jenkins -> Security_

However, since there is a dedicated `Credentials` section I was wondering whether this is outdated or I just missed something on the setup :sweat_smile:

Reply:
you're right, it's only `Credentials` 
<img width="699" alt="Screenshot 2023-09-19 at 21 35 05" src="https://github.com/ls1intum/Artemis/assets/16179620/3ce95d77-6a71-46b9-8a29-6f5c1f1d747e">